packages/sdks/src/blocks/slot/component-info.ts

```diff
@@ -0,0 +1,16 @@
+import type { ComponentInfo } from '../../types/components.js';
+
+export const componentInfo: ComponentInfo = {
+  name: 'Slot',
+  isRSC: true,
+  description:
+    'Allow child blocks to be inserted into this content when used as a Symbol',
+  docsLink: 'https://www.builder.io/c/docs/symbols-with-blocks',
+  image:
+    'https://cdn.builder.io/api/v1/image/assets%2FYJIGb4i01jvw0SRdL5Bt%2F3aad6de36eae43b59b52c85190fdef56',
+
+  // Maybe wrap this for canHaveChildren so bind children to this hm
+  inputs: [
+    { name: 'name', type: 'string', required: true, defaultValue: 'children' },
+  ],
```

Comment:
Can you confirm that you can:

- drag & drop a Slot in the editor
- add a child to it

using your latest changes?  You should be able to use an e2e server for this by toggling this: https://github.com/BuilderIO/builder/blob/f9d2be3fedfc9aedc07efef36c35066559c03446/packages/sdks/docs/DEVELOP.md#livereal-data-testing

Let me know if you need help testing this

Reply:
Sure!